mathics/builtin/list/constructing.py

```diff
@@ -185,6 +185,14 @@ class Range(Builtin):
     def apply(self, imin, imax, di, evaluation):
         "Range[imin_?RealNumberQ, imax_?RealNumberQ, di_?RealNumberQ]"
 
+        if (
+            isinstance(imin, Integer)
+            and isinstance(imax, Integer)
+            and isinstance(di, Integer)
+        ):
+            result = [Integer(i) for i in range(imin.value, imax.value, di.value)]
```

Comment:
I had considerd using `x.get_int_value()` and balked. That method name has alway felt clumsy and here, I don't see what the benefit is. If there are _x_ could be one of _several_ numeric types, then sure, having a common method name to do whatever it takes to get an int makes sense.